### PR TITLE
feat: Add default icons and general project updates

### PR DESCRIPTION
**Pull request by roman-bc-dev**
**Add the following icons to default set:**
- forward-30
- replay-30
- forward-10
- replay-10
- forward-5
- replay-5
- cast
- repeat
- shuffle
- file-download
- file-download-off
- file-download-done
- downloading

**Remove:**
- gplus

**General improvements:**
- Change .nvmrc to require `lts/gallium`
- Update dependencies to latest versions along with any associated config
- Remove lodash dependency and replace used methods in code
- Overall grunt.js code updates
- Update hbs template for index.html sample for formatting and terseness
- Update readme to describe Material Design Icon v4 inclusion as custom SVGs

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,9 +1,12 @@
 {
   "presets": [
-    ["babel-preset-env", {
-      "targets": {
-        "node": "4"
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "14"
```
Comment (misteroneill) [inline .babelrc]: Should this be `16` to match `lts/gallium` below?

Comment (roman-bc-dev) [reply .babelrc]: I think that would make sense! I'll update now.

**Review by misteroneill — APPROVED** [review]
Excellent!

**Comment by gkatsev** [comment]
Did icon codepoints change?

**Comment by gkatsev** [comment]
We shouldn't be changing codepoints of the icons, it'll break folks and require changes in video.js as well.

**Comment by misteroneill** [comment]
Good point. It hasn't been published yet, we'll make it a 4.0.

**Comment by roman-bc-dev** [comment]
The codepoints are autogenerated. Does that mean we have to keep the order of previous icons listed as we add new ones? Or are we ok with modifying them as long as we tag the change as major?

**Comment by gkatsev** [comment]
the order of the icons.json is unfortunately, important. Should be purely additive to make it non-breaking.
Unfortunately, this was probably never documented anywhere.

**Comment by gkatsev** [comment]
Fwiw, I would recommend updating it to only add the new icons at the end of the list so all the existing codepoints don't change.

**Comment by misteroneill** [comment]
Honestly, let's just delete 3.3.0 (it hasn't been `npm publish`ed) and tag 4.0.0.

**Comment by roman-bc-dev** [comment]
No, it wasn't documented. We will have to modify them if we want to remove unused icons. Like gplus in this case.

**Comment by gkatsev** [comment]
Should either do a major or not remove things 😅
Unfortunately, once we had things set up, this repo didn't get much love. One potential addition is to augment `icons.json` with the codepoint to use for the icon. Though, then the generators would need to be updated to account for that as well.

**Comment by gkatsev** [comment]
I guess webfont-generator does have a way to specify codepoints https://github.com/sunflowerdeath/webfonts-generator#codepoints

**Pull request merged**